ui/src/core_components/other/CoreReusable.vue

```diff
@@ -0,0 +1,54 @@
+<template>
+	<div class="CoreReuse">
+		<component :is="vnode" />
+	</div>
+</template>
+
+<script lang="ts">
+import { FieldType } from "../../streamsyncTypes";
+import { h, inject, watch, ref } from "vue";
+import injectionKeys from "../../injectionKeys";
+
+export default {
+	streamsync: {
+		name: "Reuse Component",
+		description:
+			"Those components are used to reuse other components. " +
+			"Reused components share the same state and are updated together.",
+		category: "Other",
+		allowedChildrenTypes: ["inherit"],
+		fields: {
+			parentId: {
+				name: "Component id",
+				type: FieldType.Text,
+				desc: "The id of the component to reuse.",
+			},
+		},
+	},
+};
+</script>
+<script setup lang="ts">
+const fields = inject(injectionKeys.evaluatedFields);
+const renderProxiedComponent = inject(injectionKeys.renderProxiedComponent);
+const vnode = ref(h("div", "No component selected"));
+
+function render() {
+	if (!fields.parentId.value) {
+		vnode.value = h("div", "No component selected");
```

Comment:
Can we make this behave like Empty _Message_ for example? Also please specify for what purpose i.e. "No component specified for reuse.".

It shouldn't show on Preview/run (use `isBeingEdited`) injectable.